internal/provider/resource_database.go

```diff
@@ -231,7 +215,7 @@ func resourceDatabaseRead(ctx context.Context, d *schema.ResourceData, meta inte
 		}
 
 		// Retry on 5XX errors
-		if resp.StatusCode() >= 500 {
+		if resp.StatusCode() >= http.StatusInternalServerError {
```

Comment:
Nice reference to http status constants.

Reply:
Thanks! I'm not a fan of "magic numbers"